Small changes to Installing Windows in the wiki (#169)
Microsoft-community/Community-website: wiki/installing-windows.md

¹ Can be usually USB HDD or your USB drive model. Your mileage may vary.

## Disk Conversion (if needed)

> The disk that you are trying to install Windows on must be a GPT disk if you are on a UEFI system or MBR if you are on a BIOS system, otherwise it will not let you install. 

Reviewer: ```suggestion
The disk that you are trying to install Windows on, must be a GPT disk if you are on a UEFI system, or MBR if you are on a BIOS system, otherwise you will get an error.
```

> Use DiskPart to change the partition scheme. ***THIS MAY DELETE ALL DATA ON THE DISK. MAKE SURE YOU HAVE BACKED UP YOUR DATA BEFORE PROCEEDING***
> 

Reviewer: ```suggestion
Use DiskPart to change the partition scheme.

::: danger CAUTION
This may **DELETE ALL DATA** on the disk! Make sure you have a backup before you proceed.
```

1. Press Shift + F10 to open a command prompt. From there, type `diskpart`.

> 2. At the DiskPart prompt, type `list disk` to list all physical disks in your computer. Note the number of the disk you're installing Windows to, and check if there is an asterisk under the GPT column. (An asterisk tells that your system disk is a GPT disk and vice versa.) If you've just switched to UEFI and do not want to delete your data on your MBR system disk, then do not proceed with DiskPart. Instead, exit the DiskPart prompt and type `mbr2gpt /convert /disk:<disk number>`.

Reviewer: ```suggestion
2. At the DiskPart prompt, type `list disk` to list all physical disks in your computer. Note the number of the disk you're installing Windows to, and check if there is an asterisk under the GPT column. (An asterisk means that your system disk is a GPT disk and vice versa.) If you've just switched to UEFI and do not want to delete your data on your MBR system disk, then do not proceed with DiskPart. Instead, exit the DiskPart prompt and type `mbr2gpt /convert /disk:<disk number>`.
```

   
  ![](./img/installing-windows/mbr2gpt.png)
   
   Otherwise, proceed to the next step.

3. Select the target disk. For example, if your system disk has the number 0, type `select disk 0`. After that, type `clean` to wipe the disk.

4. Convert the disk to the corresponding partition scheme, with either `convert mbr` or `convert gpt` to convert the disk to MBR and GPT respectively.
   
  ![](./img/installing-windows/gpt.png)

>    Exit the command prompt and continue installing.

Reviewer: ```suggestion
   Exit the command prompt and proceed with the next steps.
```

¹ Can be usually USB HDD or your USB drive model. Your mileage may vary.

## Disk Conversion (if needed)

The disk that you are trying to install Windows on, must be a GPT disk if you are on a UEFI system, or MBR if you are on a BIOS system, otherwise you will get an error.

Use DiskPart to change the partition scheme.

::: danger CAUTION
This may **DELETE ALL DATA** on the disk! Make sure you have a backup before you proceed.
1. Press Shift + F10 to open a command prompt. From there, type `diskpart`.

> 2. At the DiskPart prompt, type `list disk` to list all physical disks in your computer. Note the number of the disk you're installing Windows to, and check if there is an asterisk under the GPT column. (An asterisk means that your system disk is a GPT disk and vice versa.) If you've just switched to UEFI and do not want to delete your data on your MBR system disk, then do not proceed with DiskPart. Instead, exit the DiskPart prompt and type `mbr2gpt /convert /disk:<disk number>`.
>    
>   ![](./img/installing-windows/mbr2gpt.png)
>    
>    Otherwise, proceed to the next step.

Reviewer 2: ```suggestion
2. At the DiskPart prompt, type `list disk` to list all physical disks in your computer. Note the number of the disk you're installing Windows to, and check if there is an asterisk under the GPT column. (An asterisk means that your system disk is a GPT disk and vice versa.)

::: tip Note
If you have just switched to UEFI and do not want to delete your data on your MBR system disk, then do not proceed with DiskPart. Instead, exit the DiskPart prompt and type `mbr2gpt /convert /disk:<disk number>`.
   
![](./img/installing-windows/mbr2gpt.png)

Otherwise, you can proceed to the next step.
:::
```

¹ Can be usually USB HDD or your USB drive model. Your mileage may vary.

## Disk Conversion (if needed)

The disk that you are trying to install Windows on, must be a GPT disk if you are on a UEFI system, or MBR if you are on a BIOS system, otherwise you will get an error.

Use DiskPart to change the partition scheme.

> ::: danger CAUTION
> This may **DELETE ALL DATA** on the disk! Make sure you have a backup before you proceed.

Reviewer 2: ```suggestion
::: danger CAUTION
This may **DELETE ALL DATA** on the disk! Make sure you have a backup before you proceed.
:::
```

Use DiskPart to change the partition scheme. Note: ***THIS MAY DELETE ALL DATA ON THE DISK. MAKE SURE YOU HAVE BACKED UP YOUR DATA BEFORE PROCEEDING***

1. Press Shift + F10 to open a command prompt. From there, type `diskpart`.

2. At the DiskPart prompt, type `list disk` to list all physical disks in your computer. Note the number of the disk you're installing Windows to, and check if there is an asterisk under the GPT column. (An asterisk tells that your system disk is a GPT disk and vice versa.) If you've just switched to UEFI and do not want to delete your data on your MBR system disk, then do not proceed with DiskPart. Instead, exit the DiskPart prompt and type `mbr2gpt /convert /disk:<disk number>` instead. Otherwise, proceed to the next step.

3. Select the target disk. For example, if your system disk has the number 0, type `select disk 0`. After that, type `clean` to wipe the disk.

4. Convert the disk to the corresponding partition scheme, with either `convert mbr` or `convert gpt` to convert the disk to MBR and GPT respectively.
   
  ![](./img/installing-windows/gpt.png)

   Exit the command prompt and continue installing.

> :::

Reviewer 2: I think a separate section would be more appropriate and would be easier to link to, this is too long for a "caution".

Author: Oh, was just done with my work and I didn't read your edited comment. Makes sense, I'll also add another image to demonstrate MBR2GPT.
